sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_client_async.py

```diff
@@ -377,7 +377,8 @@ async def download_blob(self, offset=None, length=None, **kwargs):
         # type: (Optional[int], Optional[int], Any) -> StorageStreamDownloader
         """Downloads a blob to the StorageStreamDownloader. The readall() method must
         be used to read all the content or readinto() must be used to download the blob into
-        a stream.
+        a stream. The chunks() method can be used to break the content into chunks in order to process
+        them without storing the entire content in memory.
 
```

Comment:
chunks() give user an iterator which allows user to iterate the whole content in chunks. (Your English is better you can rephrase it!!)

Reply:
I guess edit file can edit this pr without pushing from local 
![image](https://user-images.githubusercontent.com/49707495/112558387-e660b480-8d8b-11eb-8bb9-5b2196bdca58.png)